src/yb/yql/pggate/util/ybc_stat.cpp

```diff
@@ -88,6 +88,9 @@ ybcstat_get_wait_event_type(uint32_t wait_event_info) {
         case YB_FLUSH_AND_COMPACTION:
             event_type = "Flush and Compaction";
             break;
+        case YB_CQL_WAIT_STATE:
```

Comment:
Should we have a separate wait event component for YCQL just like PG?
We can then have the following values
wait event component: YCQL
wait event Type: Query processing 
wait event:   Parse/Analyze/Execute

Reply:
Makes sense, I'll update it